src/utils/html-to-portable-text.ts

```
@@ -0,0 +1,535 @@
+    const captionMatch = /<figcaption[^>]*>([\s\S]*?)<\/figcaption>/.exec(figureContent)
+    const caption = captionMatch ? stripHtml(captionMatch[1]) : undefined
+    
+    // Extract attributes from audio tag
+    const srcMatch = /src="([^"]+)"/.exec(audioAttrs)
+    const hasControls = /controls/.test(audioAttrs)
+    const hasAutoplay = /autoplay/.test(audioAttrs)
+    
+    if (srcMatch && srcMatch[1]) {
+      const src = srcMatch[1]
+      const mediaRef = mediaMap.get(src)
+      const audioBlock: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)